resources/src/IconActionSearch.tsx

```diff
@@ -0,0 +1,25 @@
+import { Search } from '@geist-ui/react-icons'
+import { Keyboard, Tooltip, useToasts } from '@geist-ui/react'
+import { isIOS, isMacOs } from 'react-device-detect'
+
+const IconActionSearch = (props: { side?: boolean }) => {
+  const [, setToast] = useToasts()
+
+  const placement = props.side ? 'left' : 'top'
+  const hotkey = isIOS || isMacOs ? <Keyboard command>f</Keyboard> : <>(ctrl + f)</>
```

Comment:
Are you sure that this is not getting evaluated as `isIOS || (isMacos ? <Keyboard command> ... )`?
```suggestion
  const hotkey = (isIOS || isMacOs) ? <Keyboard command>f</Keyboard> : <>(ctrl + f)</>
```

Reply:
Those parentheses are removed when formatting with Prettier. This is fine, because the precedence of the `||` operator is higher than the one of the `? ... : ...`  (ternary) operator: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence#table